### ISSUE TITLE
Hello, if I am using a dataset with a size of 512, can I still use your pre trained model?

### ISSUE THREAD

**Issue by wangqingl** [wgcban/ChangeFormer#81]

**Comment by wusuowei666** [comment]
请问一下，有没有复现SNUNet在levir数据集上实验呢，为什么我在levir上f1分数才69呢

**Comment by wgcban** [comment]
Yes, I think you should be able to do that. But it might require good amount of GPU memory. I will try to pre-process the dataset to 256x256 images and then train on it and modify the inference code to get the final result on 512x512.


Another approach would be to down sample images by a factor of 2 and train on downsampled images. And during the testing, you can still give the downsampled image, but you can bilinear upsample the final probability map, then apply threshold to get the binary change mask. That should work fine as well.

I would try both these methods and check which one works best.

**Comment by wusuowei666** [comment]
请问一下，你有没有复现SNUNet在levir上的效果呢，看文章中复现的f1分数有88，而我复现的只有70呢，望告知，谢谢！


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "wgcban/ChangeFormer"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月22日(星期三) 晚上11:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [wgcban/ChangeFormer] Hello, if I am using a dataset with a size of 512, can I still use your pre trained model? (Issue #81)





 
Yes, I think you should be able to do that. But it might require good amount of GPU memory. I will try to pre-process the dataset to 256x256 images and then train on it and modify the inference code to get the final result on 512x512.
 
Another approach would be to down sample images by a factor of 2 and train on downsampled images. And during the testing, you can still give the downsampled image, but you can bilinear upsample the final probability map, then apply threshold to get the binary change mask. That should work fine as well.
 
I would try both these methods and check which one works best.

**Comment by wgcban** [comment]
Hey, can you pls translate that to English? Sorry about that.
